modified_slot_machine.R

```diff
@@ -0,0 +1,71 @@
+#g et the symbols
+get_symbols <- function(n_wheels = 3) {
+  wheel <- c("DD", "7", "BBB", "BB", "B", "C", "0")
+  sample(wheel, size = n_wheels, replace = TRUE, 
+         prob = c(0.03, 0.03, 0.06, 0.1, 0.25, 0.01, 0.52))
+}
+
+calculate_score <- function(wheel_symbols){
+  stopifnot(is.vector(wheel_symbols))
```

Comment:
This one I'm totally lost! Ahaha 

stopifnot(is.vector) = stop if the result from (whee;l_symbols) is not a vector? What does that being a vector help me with?

Reply:
Just an introduction to another aspect of functions; you can set criteria that need to be met for the function to continue. Think of it as a proactive measure; you know what needs to go into a function to make it work, can you define that a bit in the beginning to limit issues down the line (like silent errors).

Try running

`calculate_score(mtcars)`

Since it is not a vector, it will return an error. This stopifnot ought to be expanded to additional conditions, since even something like c("A") is a vector. Down the line you can actually have these report custom error messages that can be informative.